argparse/templates/cwltool.j2

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env cwl-runner
+# This tool description was formed automatically by argparse2cwl ver. {{ version }}
+# To form again: $ {{ formcommand }} --generate_cwl_tool
+# Help: $ {{ formcommand }} --help_arg2cwl
```

Comment:
The goal was to remind a user how to get a list of all `argparse2cwl` options because a default `-h` flag will display tool's own help message.